elfloader-tool/src/arch-arm/drivers/smp-psci.c

```diff
@@ -3,6 +3,7 @@
  *
  * SPDX-License-Identifier: GPL-2.0-only
  */
+#include <autoconf.h>
```

Comment:
We should include `autoconf.h` in `elfloader_common.h`, then we don't have to do this in every file explicitly.